skin/src/main/java/org/researchstack/skin/UiManager.java

```diff
@@ -71,6 +71,7 @@ public static UiManager getInstance()
      * @param context android context
      * @return a Step used for Eligibility within the onboarding process
      */
+    @Deprecated
```

Comment:
For documentation purposes, could you  link to the new method in ResourceManager?